Example/AztecExample.xcodeproj/xcshareddata/xcschemes/AztecExample.xcscheme

```diff
@@ -49,7 +49,7 @@
             </BuildableReference>
          </TestableReference>
          <TestableReference
-            skipped = "NO">
+            skipped = "YES">
```

Comment:
Can you revert this change please?

The UI tests have been disabled in a separate PR, and this change doesn't really seem to prevent them from running anyway.